kedro/config/omegaconf_config.py

```diff
@@ -302,6 +307,12 @@ def _is_valid_config_path(self, path):
             ".json",
         ]
 
+    @staticmethod
+    def _register_new_resolvers(resolvers: dict[str, Callable]):
+        """Register custom resolvers"""
+        for name, resolver in resolvers.items():
+            OmegaConf.register_new_resolver(name, resolver, replace=True)
+
```

Comment:
Does it generates any log when a resolve is replaced?

Reply:
Why do we want to override built-in resolver? Maybe I am missing some context in the issue.

Reply:
I think we decided in the tech design to have this flag set to true so users have the option to overwrite the built-in resolvers.
No logs generated, I think it might show up twice per resolver because config loader is initialised twice during one run (once when logging is set up and once again when context is created)

Reply:
I think it would be nice to produce some log here. Either via OmegaConf builtin logging (if any) or we should have log when it is overwritten.

I think it should be an INFO level log because normally you won't overwrite a built-in resolver. Is there any risk if one want to overwrite built-in?

However, if it is producing duplicate logs, maybe it should be a DEBUG level.  

Note that Logging no longer need config_loader, see https://github.com/kedro-org/kedro/pull/2642 This is merged into `develop` only (0.19)